armi/bookkeeping/db/database.py

```diff
@@ -1394,8 +1394,9 @@ def getHistories(
                 # current time step and something has created the group to store aux data
                 continue
 
-            cycle = h5TimeNodeGroup.attrs["cycle"]
-            timeNode = h5TimeNodeGroup.attrs["timeNode"]
+            # might save as int or np.int64, so forcing int keeps things predictable
+            cycle = int(h5TimeNodeGroup.attrs["cycle"])
+            timeNode = int(h5TimeNodeGroup.attrs["timeNode"])
```

Comment:
I couldn't root cause the sometimes-`np.int64` issues with the time steps, so I added a forced int here during `getHistory`